README.md

```diff
@@ -5,6 +5,16 @@
 
 This repository contains the beta-stage publishing component of data.gov.uk.
 
+# Deployment
+
+Continuous Integration has been setup using github actions. 
+Tests are run on pull requests, deployments to Staging are via merges of branches into the `main` branch.
+In order to carry out a release to production a developer in the govuk team will need to create a release tag that leads with a `v` and approve of the deployment in github actions.
+
+Further information about the deploying to PaaS are in the developer documents here: 
+
+https://docs.publishing.service.gov.uk/manual/data-gov-uk-deployment.html#paas-staging-and-production-environments
```

Comment:
Out of scope, but could the contents of that document be brought into the `docs/` folder of this repo? That way we can keep all the documentation here in one place (helping it stay up to date), and it will also be auto-imported into the Developer Docs.

Reply:
I think the reason it's there is so that there's only one place where that information needs to be updated.